### PR TITLE
Integrated run.py commands with flask-script and moved it to manage.py

### PR DESCRIPTION
**Pull request by RaghavPro**
Using command line arguments this time instead of raw_input

Details mentioned in #109

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -9,8 +7,55 @@
...
+    from cleansweep.loaddata import main
+    main(directory)
+
+
+@manager.command
+def loadfile(filename):
+    "Loads data from the specified file."
+
+    from cleansweep.loaddata import main_loadfiles
+    main_loadfiles(filename)
```
Comment (anandology) [inline manage.py]: I think main_loadfiles takes a list of filenames, just a single filename.

File: manage.py [hunk 2]
```diff
@@ -9,8 +7,55 @@
...
+    from cleansweep.core.mailer import run_worker
+    run_worker()
+
+
+@manager.command
+def update_parents(place):
+    "Updates parents of all places below the specified key."
+
+    from cleansweep.loaddata import update_parents
+    update_parents(place)
```
Comment (anandology) [inline manage.py]: update_parents was a hack added to fix some issue on production. Please drop that from manage.py.

File: manage.py [hunk 3]
```diff
@@ -9,8 +7,55 @@
...
+
+
+if __name__ == '__main__':
+    manager.run({'db': MigrateCommand})
```
Comment (anandology) [inline manage.py]: Please use:

    manager.add_command('db', MigrateCommand)

on the top instead of passing it as argument to `manager.run`.

**Comment by anandology** [comment]
Please update the Readme as well to reflect these changes.

File: manage.py (hunk 1 above)
Comment (RaghavPro) [inline manage.py]: Whoops. Will fix that. I'm supposed to create a new method for loading a single file, right?

**Pull request merged**